src/sentry/api/serializers/models/organization_member/base.py

```diff
@@ -44,6 +44,9 @@ def get_attrs(
         users_by_id: MutableMapping[str, Any] = {}
         email_map: MutableMapping[str, str] = {}
         for u in user_service.serialize_many(filter={"user_ids": users_set}):
+            # Filter out the emails from the user data
+            if "emails" in u:
+                del u["emails"]
```

Comment:
`u.pop("emails", None)` is the simpler way to do this

Reply:
🤦 So much simpler. Thank you!